doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -87,23 +87,23 @@ To ensure alignment with the infrastructure profile catalogue, the following req
 | e.cap.014/ infra.com.acc.cfg.004 | Hardware coprocessor support (GPU/NPU) | Compute Intensive GPU | | |
 | e.cap.016/ infra.com.acc.cfg.005 | FPGA/other Acceleration H/W | Firmware-programmable adapter | | |
 
+<p align="center"><b>Table 2-1a:</b> Cloud Infrastructure Software Profile Extensions Requirements for Compute</p>
```

Comment:
@pgoyal01 To keep the scheme you've adopted for tables' numbering, I think this table should be 2-1b and the previous table should be 2-1a rather than 2-1